### DISCUSSION TITLE
wallet-connect wallet with AA ERC-4337 integration with Dapps ( ex. Uniswap )

### DISCUSSION THREAD

**Discussion by ace3** [WalletConnect/walletconnect-monorepo#2590] [Developer Support] [answered]
Hi, i want to make a wallet app with AA integration, then in order to connect to Dapps, I will use Wallet Connect.

Could you please let me know if this is doable?
bridge the connection from the wallet app to Dapps ( using wallet-connect )
intercept the request to approve/ swap from Dapps ( example: uniswap )
transform the generic signed transaction created by uniswap using userop.js by stackup
send it as userOperation

thank you.

**Comment by finessevanes** [comment, marked as answer]
Hey - check out our AA docs: https://docs.walletconnect.com/2.0/web/sign/smart-wallets

**Reply by ace3** [reply]
Hi, thank you for the response. I don't realize that there is an update on the documentation.

thanks again.

**Reply by finessevanes** [reply]
Np - thanks for asking!

**Reply by bobbygggard** [reply]
Good luck with your app!

**Reply by ya-ni-na** [reply]
Really wish you same

**Reply by JaffaSweet** [reply]
good luck good idea

**Reply by artemni1982** [reply]
thanks for the help, it's clear now

**Reply by meta1pony** [reply]
Thank you very much!

**Reply by endrews21** [reply]
great idea!

**Reply by AkimovAndre** [reply]
It's not easy, my friends...

**Reply by ErmilovVlad** [reply]
great

**Reply by AHMED-UUU** [reply]
hay

**Reply by coinku13212** [reply]
good

**Reply by AnnaBirzk** [reply]
everything clear

**Reply by atlanticgg** [reply]
clear, thanks

**Reply by jendos71** [reply]
clear

**Reply by hedera1** [reply]
hey

**Reply by prsm171** [reply]
Hi !!! Good.

**Reply by ErmilovVlad** [reply]
nice

**Reply by KasaevCrypto** [reply]
Hi !!! Good.

**Reply by robinap5** [reply]
Thanks!

**Reply by AndreyPetrov74** [reply]
oh thank you

**Reply by Luchia777** [reply]
thank you!

**Reply by Ellu39940** [reply]
A lot of thanks.

**Reply by Solnce55** [reply]
thanks

**Reply by Jim555122** [reply]
Thanks!

**Reply by Johnmay5** [reply]
Thanks! Great

**Reply by Linaderss** [reply]
Thanks!

**Reply by TelemetryTemplar** [reply]
I hope everything works out.

**Reply by Yevhenii2209** [reply]
Thank you!

**Reply by wibowok07** [reply]
Thank you!

**Reply by matthewftcash** [reply]
Thanks)

**Reply by Ky3bMu4b1** [reply]
very good

**Reply by Evgenyweb** [reply]
Thanks)

**Comment by nadin29091** [comment]
thx! rly!)

**Comment by shtolsevgeny** [comment]
Super!

**Reply by Eldaddr** [reply]
Cool

**Reply by gorbunova13** [reply]
really cool!

**Comment by Ramil08061966** [comment]
Great

**Comment by vestayson** [comment]
Have a nice day everyone

**Reply by AkimovAndre** [reply]
you too

**Comment by artemni1982** [comment]
Ik denk geen probleem

**Comment by valenl1956** [comment]
Pretty useful!

**Comment by Durnoyz** [comment]
Hey, thanks!

**Comment by 1tomis** [comment]
Great!

**Comment by memichelleva** [comment]
nice!)

**Comment by valensov** [comment]
Good idea!

**Reply by IraChek** [reply]
What are you talking about? where's the idea?

**Comment by ARTNEON69** [comment]
Great!

**Comment by Dennise93293222** [comment]
gooood

**Comment by marya331** [comment]
Super!!

**Comment by Lissett89716587** [comment]
Oh, good!

**Comment by Starla61422457** [comment]
Рerfect

**Comment by ag2silver** [comment]
Useful for sure

**Comment by AllaAliona** [comment]
Nice!

**Comment by VVDonny** [comment]
Great! Thanks

**Comment by tonettiancartinson** [comment]
I like it

**Comment by MichaelN5** [comment]
yeaaa

**Comment by nailatunas** [comment]
Have a nice day everyone

**Comment by listeus** [comment]
Useful information, thx! :)

**Comment by Alesio22** [comment]
I hope data security will be the main thing in the wallet!?

**Comment by Utkautkaut** [comment]
Great!!!

**Comment by Kaligora** [comment]
thx!  rly!)

**Comment by QWERTYJES** [comment]
thx! rly!)

**Comment by artempesok** [comment]
thanks, i like it

**Comment by TheRowBack** [comment]
Thank you

**Comment by UstinovKarl** [comment]
Need to try)

**Comment by AHMED-UUU** [comment]
Thank you

**Comment by Alexandrosasa** [comment]
Great!

**Comment by Mrazzz2** [comment]
Wooow nice)

**Reply by TerryPShepherd34** [reply]
It's a promising opportunity

**Comment by Roarkiii** [comment]
Good

**Comment by Krid1111** [comment]
Good news

**Comment by Nikopiter** [comment]
Very good proposal

**Comment by AllaAliona** [comment]
Well!

**Comment by bodya80023** [comment]
Good idea

**Comment by 01serg2233** [comment]
Great!

**Comment by willilindseylynch** [comment]
good idea, and usefull

**Comment by gorbunova13** [comment]
thanks

**Comment by oliperroni** [comment]
Thanks a lot

**Comment by Rampon11** [comment]
Perfect

**Comment by Rusgreat** [comment]
Incredible!

**Comment by Crystall0** [comment]
Great! Thank you 🫶🏻💕

**Comment by KalinaXolo** [comment]
Nice!

**Comment by Kasian8702** [comment]
Good luck

**Comment by HarmonG5** [comment]
Hello! That's great to hear. If you're open to suggestions or discussions, feel free to share more about the context or topic. This way, we can provide more accurate and beneficial input.

**Reply by KalinaXolo** [reply]
Thank you for the invitation to the discussion!

**Comment by tanushshshka1** [comment]
beautiful

**Comment by dimashulll** [comment]
good luck

**Comment by Variant101** [comment]
amazing

**Comment by Timma13** [comment]
Very Nice!

**Comment by ethanjamesA** [comment]
Thanks very much

**Comment by CodeCrusader911** [comment]
Have a nice day everyone.

**Comment by rekash13** [comment]
Hi. Great!

**Comment by Starker5** [comment]
HI ok.not bad

**Comment by Dilimed** [comment]
All great!

**Comment by Khurshidaa** [comment]
Not bad

**Comment by Eujynot** [comment]
tnx) rly!

**Comment by rdeboer081** [comment]
perhaps
Maybe

**Comment by arturbebov45** [comment]
lol this is good

**Comment by Burmalda121** [comment]
good job

**Comment by roosp8388** [comment]
information for thought

**Comment by BritvaBo** [comment]
good question!

**Comment by Evgenyweb** [comment]
Thanks)

**Comment by zlatok888** [comment]
great)

**Comment by Evgenyweb** [comment]
cool

**Comment by alisa4s** [comment]
popular

**Comment by sofia7774** [comment]
nice

**Comment by polina345v** [comment]
klass

**Comment by pozhitokroman** [comment]
We need to close the thread. Spam...
